### DISCUSSION TITLE
I found that 'Noto_Sans_KR' font at @next/font/google doesn't have 'korean' subset.

### DISCUSSION THREAD

**Discussion by engadori** [vercel/next.js#45035] [Help] [answered]
In @next/font version 13.1.2, There is no 'korean' subset in 'Noto_Sans_KR' font.

When I set the subset 'latin', It doesn't work in korean font.

is it a bug?

**Comment by hanneslund** [comment, marked as answer]
Korean font files aren't available to be preloaded. This is because the font family contains 100+ font files. In your case you can disable preloading to get it to work.
```js
import { Noto_Sans_KR } from "@next/font/google";
const notoSansKR = Noto_Sans_KR({
  weight: "400",
  preload: false,
  adjustFontFallback: false // can try with `true` and `false` and see which results in the least layout shift for you
})
```

**Reply by engadori** [reply]
It works. thanks.
